appengine/accessible/workspaceview.component.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Blockly Games: Accessible
+ *
+ * Copyright 2016 Google Inc.
+ * https://github.com/google/blockly-games
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how a Blockly.Workspace is
+ * rendered in AccessibleBlockly. Modified to run music code.
```

Comment:
Woah -- I don't understand why this is needed, especially if the AccessibleBlockly library is supposed to work generally with external dev use cases.

Is this because of runCode()? If so, then I'm wondering if runCode() ought to live outside blocklyApp.WorkspaceView instead, and be something that external developers implement. You shouldn't force devs to monkeypatch your library in order to use it, and I'm not sure it needs to be part of your workspace anyway, if you just define the workspace as the canvas where they create blocks. E.g. in Maze, the 'run code' etc. stuff (as well as the visualization) is off to the side, outside the main workspace.